<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor</class>
 <widget class="QDialog" name="monitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>热仿真数据处理</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>551</width>
     <height>501</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>6</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>7</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>8</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>9</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>10</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>11</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>12</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>13</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>14</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>15</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>16</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>17</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>18</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>X坐标</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y坐标</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Z坐标</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>温度</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>元器件类型</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="1" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="2" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="3" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="4" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="5" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="6" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="6" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="6" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="6" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="7" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="7" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="7" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="7" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="8" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="8" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="8" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="8" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="9" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="9" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="9" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="9" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="10" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="10" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="10" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="10" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="11" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="11" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="11" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="11" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="12" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="12" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="12" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="12" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="13" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="13" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="13" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="13" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="14" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="14" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="14" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="14" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="15" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="15" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="15" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="15" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="16" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="16" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="16" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="16" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="17" column="0">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="17" column="1">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="17" column="2">
    <property name="text">
     <string/>
    </property>
   </item>
   <item row="17" column="3">
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
